### DISCUSSION TITLE
Can we set an upper limit of usage for budget control?

### DISCUSSION THREAD

**Discussion by naeem-qv** [vercel/community#6850] [Feature Question] [answered]
### Question

I am interested in learning more about the monitoring and budget control options you offer. Specifically, we are looking to set a strict monthly budget limit of $100 for our projects. Once this limit is reached, we would like our projects to be taken offline to avoid any additional charges. Is it possible to implement such a feature within your platform?

**Comment by swarnava** [comment, marked as answer]
Hi,

You can use our [Spend Management](https://vercel.com/docs/pricing/spend-management) feature to achieve this.
